pkg/controllers/vrep/vdbverify_reconciler_test.go

```diff
@@ -31,25 +31,26 @@ import (
 func testIncompatibleDB(ctx context.Context, sourceVersion, targetVersion string,
 	sourceUsingVclusteropsDeployment bool, expectedReason string,
 	expectedStatusConditionValue bool, expectedState string) {
-	sourceVDBName := v1beta1.MakeSourceVDBName()
-	sourceVDB := vapi.MakeVDB()
-	sourceVDB.Name = sourceVDBName.Name
-	sourceVDB.Namespace = sourceVDBName.Namespace
+	sourceVdbName := v1beta1.MakeSourceVDBName()
+	sourceVdb := vapi.MakeVDB()
+	sourceVdb.Name = sourceVdbName.Name
+	sourceVdb.Namespace = sourceVdbName.Namespace
 	if sourceUsingVclusteropsDeployment {
-		sourceVDB.Annotations[vmeta.VClusterOpsAnnotation] = vmeta.VClusterOpsAnnotationTrue
+		sourceVdb.Annotations[vmeta.VClusterOpsAnnotation] = vmeta.VClusterOpsAnnotationTrue
 	}
-	sourceVDB.Annotations[vmeta.VersionAnnotation] = sourceVersion
-	test.CreateVDB(ctx, k8sClient, sourceVDB)
-	defer test.DeleteVDB(ctx, k8sClient, sourceVDB)
-
-	targetVDBName := v1beta1.MakeTargetVDBName()
-	targetVDB := vapi.MakeVDB()
-	targetVDB.Name = targetVDBName.Name
-	targetVDB.Namespace = targetVDBName.Namespace
-	targetVDB.Annotations[vmeta.VClusterOpsAnnotation] = vmeta.VClusterOpsAnnotationTrue
-	targetVDB.Annotations[vmeta.VersionAnnotation] = targetVersion
-	test.CreateVDB(ctx, k8sClient, targetVDB)
-	defer test.DeleteVDB(ctx, k8sClient, targetVDB)
+	sourceVdb.Annotations[vmeta.VersionAnnotation] = sourceVersion
+	test.CreateVDB(ctx, k8sClient, sourceVdb)
+	defer test.DeleteVDB(ctx, k8sClient, sourceVdb)
+
+	targetVdbName := v1beta1.MakeTargetVDBName()
+	targetVdb := vapi.MakeVDB()
+	targetVdb.Name = targetVdbName.Name
+	targetVdb.Namespace = targetVdbName.Namespace
+	targetVdb.Annotations[vmeta.VClusterOpsAnnotation] = vmeta.VClusterOpsAnnotationTrue
+	targetVdb.Annotations[vmeta.VersionAnnotation] = targetVersion
```

Comment:
@spilchen should we conform to this version requirement in official doc? Does that mean the minimum version for source should be 24.3.0 and minimum version for target be 24.4.0?

Reply:
Yes, I guess we should conform to it. The minimum version for target should be 24.3.0 though. Source and target can be the same version. 

@chinhtranvan what is the major version in the context of Vertica's new versioning strategy? Is it the year component? i.e. if source is 24.3.0, target can be at most 25.3.0?

I am wondering if the vdb verifier should check that we are at most 1 major version away.

Reply:
I just have a discussion with Alex, He mentioned that "target >= source should be the only criterion. No specific version requirements."

Reply:
yeah I think internally the real implementation would handle it as long as target version is higher or equal (it might still work even if target is lower but that's too rare and certainly not guaranteed), it's just that in the official documentation we are tightening the constraints

Reply:
We need to check the source is at least at 24.3.0. Otherwise, we won't have support in vcluster for replication. And the target should >= source.

Reply:
Then, you need to update the target version to be higher or equal to the source version. I've already asked Monica to update the docs